### ISSUE TITLE
[Quest][Classic] Uniting the Shattered Amulet

### ISSUE THREAD

**Issue by Golganeth** [dalaranwow/dalaran-wow#500]
http://wotlk.openwow.com/quest=3627

**Issue**: The provided weapons are not weakening the mobs after using them. This makes the quest really hard to complete if you are not a high level.

**How it should work:** After equipping the weapon and using it on each of the mobs, their damage should get reduced considerably.

---
- **Enchanted Azsharite Felbane Sword -** http://wotlk.openwow.com/item=10696
- **Enchanted Azsharite Felbane Dagger -** http://wotlk.openwow.com/item=10697
- **Enchanted Azsharite Felbane Staff -** http://wotlk.openwow.com/item=10698

**Comment by n-ghost** [comment]
Currently the quest is on auto-complete.

There is another issue: all 3 elites won't drop the amulets required to complete the quest in blizzlike way. 

Grol (http://www.wowhead.com/npc=7665/) should drop http://www.wowhead.com/item=10753

Allistarj (http://www.wowhead.com/npc=7666/) should drop http://www.wowhead.com/item=10755

Sevine (http://www.wowhead.com/npc=7667/) should drop http://www.wowhead.com/item=10754/

The quest should become completed only after the player gathers all 3 of the amulets.

**Comment by Daveonx** [comment]
Currently none of the items obtained from the quest [Enchanted Azsharite Fel Weaponry](http://www.wowhead.com/quest=3625/enchanted-azsharite-fel-weaponry) and the [Fel Salve](http://www.wowhead.com/item=11582/fel-salve) are weakening the the three elites from the quest. Regardless they can be killed and all three drop the needed amulets for the quest.

**Comment by Rushor** [comment]
this effect should applied on the mob once hit by the item: http://wotlk.openwow.com/spell=12941

**Comment by Rushor** [comment]
DB/Quest: Uniting the Shattered Amulet - Add 'Fel Curse Effect' for Grol the Destroyer, Archmage Allistarj and Lady Sevine

* closes https://github.com/dalaranwow/dalaran-wow/issues/500
* commit: 62fb943bb64222e1d74ddb972158d8978878943d

![GitHub Logo](http://i.imgur.com/QCloboK.jpg)
![GitHub Logo](http://i.imgur.com/Nz0jhl3.jpg)
![GitHub Logo](http://i.imgur.com/xT8xwly.jpg)

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
